cli/tests/e2e/test_output.py

```diff
@@ -504,3 +504,17 @@ def test_sca_lockfile_only_output(run_semgrep_on_copied_files: RunSemgrep, snaps
         results,
         "results.txt",
     )
+
+
+@pytest.mark.kinda_slow
```

Comment:
please make it pass also in osemgrep.
You need to add the relevant code in osemgrep and add an @pytest.mark.osempass here also.

Reply:
Sorry @aryx this is pass due and I have a very full week. Is it possible that I go back and reimplement this work next week after the public demo?